browser/src/VariantPage/__snapshots__/VariantPage.spec.tsx.snap

```diff
@@ -8061,7 +8061,7 @@ exports[`VariantPage with the dataset "gnomad_r3" has no unexpected changes 1`]
                           dy="0em"
                           x={-35}
                         >
-                          SiteQuality
+                          log(SiteQuality)
```

Comment:
looks like `log(whatever)` got interpreted as a string literal rather than getting evaluated and interpolated

Reply:
If I'm not mistaken I think for the chart labels we might not want to evaluate but display as a log of the metric like "log(SiteQuality)" directly as a label, @ch-kr is that the display we wanted?

Reply:
oh right sorry, wrong context